src/main/java/pw/chew/chanserv/listeners/UwUChannelHandler.java

```diff
@@ -5,7 +5,7 @@
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
 
 public class UwUChannelHandler extends ListenerAdapter {
-    private final String UWU_REGEX = "[\\.|?|!|~]";
+    private final String UWU_REGEX = "[\\.|?|!|~|\\-|_|\\*]";
```

Comment:
```suggestion
    private final String UWU_REGEX = "[\\.|?|!|~|\\-|_|*]";
```

Reply:
We need to escape the * character since that is a valid regex character

Reply:
Testing with regex101 shows it needs to be 
```suggestion
    private final String UWU_REGEX = "[\.|?|!|~|\-|_|\*]";
```